string param
string value 	#TODO change to int8
---
bool success